common/changes/contextualmenu-fixscrollbar-ie_2017-01-05-01-42.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "changed ms-contextualmenu-link from inline-block to block to prevent scrollbars from appearing incorrectly",
```

Comment:
Can you make this publishable changelog text? Example

ContextualMenu: tweaked css to prevent scrollbars from appearing in IE